.circleci/config.yml

```diff
@@ -512,7 +512,9 @@ jobs:
           name: deploy
           command: |
             if [ $AZURE_STORAGE_SAS_TOKEN ]; then
-              azcopy copy "/tmp/verdaccio_dist/*/verdaccio/*" "https://reactspectrum.blob.core.windows.net/reactspectrum/$CIRCLE_SHA1/verdaccio${AZURE_STORAGE_SAS_TOKEN}" --recursive
```

Comment:
huh, wildcards definitely show up in their docs. wonder what's different here: https://learn.microsoft.com/en-us/azure/storage/common/storage-ref-azcopy-copy